RadarSDK/RadarNotificationHelper.m

```diff
@@ -136,7 +136,6 @@ + (void)removePendingNotificationsWithCompletionHandler:(void (^)(void))completi
         NSMutableArray *identifiers = [NSMutableArray new];
         for (UNNotificationRequest *request in requests) {
             if ([request.identifier hasPrefix:kSyncGeofenceIdentifierPrefix]) {
-                [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:[NSString stringWithFormat:@"Found pending notification to remove | identifier = %@", request.identifier]];
```

Comment:
intentional?

Reply:
This isn't related to this (but intentional). I can take it to keep it atomic.